### ISSUE TITLE
[Bug]: [Nightly]Nightly test has taken more time on average than before and sometimes failed for timeout 

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#23406]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 43a9e17
- Deployment mode(standalone or cluster):standalone
- MQ type(rocksmq, pulsar or kafka):    rocksmq
- SDK version(e.g. pymilvus v2.0.0rc2):2.4.0.dev7
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Nightly test has taken more time on average than before and sometimes failed for timeout.
Milvus not panic.
Latest:
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/340/pipeline/123

### Expected Behavior

work as before

### Milvus Log

[artifacts-milvus-standalone-nightly-340-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Milvus%20Nightly%20CI/job/master/340/artifact/artifacts-milvus-standalone-nightly-340-pymilvus-e2e-logs.tar.gz)

**Comment by NicoYuan1986** [comment]
The situation seems to start on April 10th.

**Comment by binbinlv** [comment]
It continuously reproduced since 4/9, so set it as urgent.

**Comment by binbinlv** [comment]
Nightly on this commit: 3c52d76, it runs ended in 1h34m
Nightly on this commit: d85f673, it started to ended in 3h48m.


The commits between 3c52d76 and d85f673 are:
![uSNzmXb1cF](https://user-images.githubusercontent.com/83755740/231677487-2c5dcd1b-37e6-4739-9906-0cb247b4da3c.jpg)

so @longjiquan  could you help to have a look first? Thanks.

**Comment by binbinlv** [comment]
2.2.0 branch:
Nightly on this commit: dfee106, it runs ended in 1h 30m,
Nightly on this commit:7e0f3e9,  it runs ended in 3h 15m
The commits between dfee106 and 7e0f3e9 are:
![pdxh8ElIwK](https://user-images.githubusercontent.com/83755740/231685548-de2f88f1-f133-4eef-94f1-3748d59b6148.jpg)

**Comment by binbinlv** [comment]
So from the master and 2.2.0 branch, the suspicious commit may be "Change balance check interval to 10s",
 
@xiaofan-luan could you please have a look either? Thanks.

**Comment by xiaofan-luan** [comment]
I will take a look into it

**Comment by NicoYuan1986** [comment]
Update: kafka and standalone failed for timeout.
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/346/pipeline/155

**Comment by NicoYuan1986** [comment]
Update: kafka and standalone failed for timeout.
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/348/pipeline/155

**Comment by sunby** [comment]
@binbinlv @NicoYuan1986  Maybe we can change the checkInterval back to 1s and retry. This params control all checkers' interval and may lead to slow collection loading.

**Comment by binbinlv** [comment]
Will verify through tonight's nightly.

**Comment by binbinlv** [comment]
@sunby 

Verified, Nightly time of the master branch decreased, fixed.

But 2.2.0 branch is still not OK, I think 2.2.0 has the same issue, could you please fix it on 2.2.0 branch either?
Thanks.

**Comment by xiaofan-luan** [comment]
> /assign @binbinlv

I think it's probably not a good idea to check every 1s if there are no events(nodes up/down) because it might takes too much cpus on a cluster with many collections. Thoughts?

**Comment by sunby** [comment]
> > /assign @binbinlv
> 
> I think it's probably not a good idea to check every 1s if there are no events(nodes up/down) because it might takes too much cpus on a cluster with many collections. Thoughts?

Checkers will be executed in one goroutine sequentially. So IMHO it will not take too much cpu.
Is the purpose of changing this parameter to 10s to fix a certain issue?

**Comment by xiaofan-luan** [comment]
> > > /assign @binbinlv
> > 
> > 
> > I think it's probably not a good idea to check every 1s if there are no events(nodes up/down) because it might takes too much cpus on a cluster with many collections. Thoughts?
> 
> Checkers will be executed in one goroutine sequentially. So IMHO it will not take too much cpu. Is the purpose of changing this parameter to 10s to fix a certain issue?

No, just because log  are printed too often and user see cpu usage to be high on 2c4g machines

**Comment by congqixia** [comment]
see also #23870
Shall be fixed by #23925 #23928 
@binbinlv

**Comment by binbinlv** [comment]
The time for nightly decreased to the previous normal time, fixed, and closed.
